yt/chyt/controller/internal/api/helpers.go

```diff
@@ -92,6 +93,30 @@ func transformToStringSlice(value any) (any, error) {
 	return transformedAttributes, nil
 }
 
+func transformFilterValues(value any) (any, error) {
+	if value == nil {
+		return nil, nil
+	}
+
+	filterVals, ok := value.(map[string]any)
+	if !ok {
+		typeName := reflect.TypeOf(value).String()
+		return nil, unexpectedTypeError(typeName)
+	}
+
+	newMap := make(map[string]any, len(filterVals))
+	for k, v := range filterVals {
+		if k == "state" {
+			newMap[k] = strawberry.OpletState(v.(string))
```

Comment:
Need to validate the type of the `v` parameter before accessing it with `.(string)`
I'll fix it during the import